package.json

```diff
@@ -15,6 +15,7 @@
 		"@salesforce/ts-types": "^2.0.9",
 		"@types/node": "^17",
 		"ansis": "^3.2.0",
+		"globby": "^11",
```

Comment:
Globby v12+ is pure ESM, which was causing problems since we're CJS. So I'm using v11, which is the same version that v4 of the scanner is already using.